### PR TITLE
Add seeding status

### PR DESCRIPTION
**Pull request by Audionut**
https://ptpimg.me/ydjdp5.png

Before I get too caried away with this, what say thee.
Icon color is just standout, I'll defer to any suggestion.

Obviously not every site would be supported.

### CODE REVIEW

**Comment by Purfview** [comment]
Interesting idea.

>Obviously not every site would be supported.

Why not? I guess you mean that for this to work you need `positiveMatch`? Maybe `positiveMatch2` could be introduced?

**Comment by Audionut** [comment]
From experience with that sites script I maintain, not every site returns seeding status, either/or HTML or API.

The only thing I've done so far is see if I could actually make it work. I have done exactly zero testing save for that screenshot output. Although, I did have icons on one line first, and the seeded icon was first. Not sure if that was because of the order list, or because it's else if before else 'found'.

Presumably I could make it an option. I'm on mobile now, but presumably I could add an option check before the else if and assign icon as a seeding match.

**Comment by Purfview** [comment]
>From experience with that sites script I maintain, not every site returns seeding status, either/or HTML or API.

That I understand, I thought that you meant it returns status but it wouldn't be supported.

>I did have icons on one line first, and the seeded icon was first..

That depends from order in `const valid_states`.

>Presumably I could make it an option. I'm on mobile now, but presumably I could add an option check before the else if and assign icon as a seeding match.

Make a post there, if noone complains then no need for option, I think.

**Comment by Audionut** [comment]
Apart from waiting for that feedback, I don't think I can do much more with this. No other site I have access too has a seeding element available to hook onto.

If this gets pushed, would you like me to create an issue outlining how to find and add the regex match?

**Comment by Purfview** [comment]
>If this gets pushed, would you like me to create an issue outlining how to find and add the regex match?

Eventually it will be, yeah you can create it [at show and tell](https://github.com/Purfview/IMDb-Scout-Mod/discussions/categories/show-and-tell) discussion, I'll link it in the PR "guide".

**Comment by Purfview** [comment]
> I don't think I can do much more with this.

There is still one pending review.

**Comment by Audionut** [comment]
> There is still one pending review.

I don't see a review anywhere.

**Comment by Audionut** [comment]
Nope. If you do actually mean a review comment on some line/s of code, it does have to be confirmed, I think top right hand corner, after making a comment at a line of code. It's a bit clunky how that works.

**Comment by Purfview** [comment]
Can you make a screenshot what is there for you?

**Comment by Audionut** [comment]
https://ptpimg.me/7lz2ru.png

**Review by Purfview — CHANGES_REQUESTED** [review]

**Comment by Purfview** [comment]
Did it appeared now?

File: IMDb_Scout_Mod.user.js [hunk 1]
```diff
@@ -6926,7 +6926,7 @@ function addLink(elem, site_name, target, site, state, scout_tick, post_data) {
                                                                             : icon.css('border-color', 'rgb(230,200,100)');
     } else if (state == 'seeding') {
       (getPageSetting('highlight_sites').split(',').includes(site['name'])) ? icon.css('border-color', 'rgb(255,0,255)')
-                                                                            : icon.css('border-color', 'rgb(100,200,230)');
+                                                                            : icon.css('border-color', 'rgb(130,0,130)');
```
Comment (Audionut) [inline IMDb_Scout_Mod.user.js:6929]: Something like this?

Comment (Purfview) [reply IMDb_Scout_Mod.user.js:6929]: Yeah

**Comment by Audionut** [comment]
> Eventually it will be, yeah you can create it at [show and tell](https://github.com/Purfview/IMDb-Scout-Mod/discussions/categories/show-and-tell) discussion, I'll link it in the PR "guide".

Added: https://github.com/Purfview/IMDb-Scout-Mod/discussions/240

I'll correct the link after merge so I can delete the branch.

**Pull request merged**